<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>SolverCfgWidget</class>
<widget class="SolverCfgWidget" name="solverCfgWidget">

<layout class="QVBoxLayout">

<item>
<layout class="QHBoxLayout">
	<item>
	<widget class="QLineEdit" name="scrambleEditor">
		<property name="placeholderText">
		<string>Enter your scramble here...</string>
		</property>
	</widget>
	</item>

	<item>
	<widget class="QPushButton" name="solveButton">
		<property name="text">
		<string>Solve!</string>
		</property>
	</widget>
	</item>
</layout>
</item>

<item>
	<layout class="QHBoxLayout">
		<item>
		<widget class="QLabel">
			<property name="text">
			<string>Min moves</string>
			</property>
		</widget>
		</item>

		<item>
		<widget class="QLineEdit" name="minMovesEditor">
			<property name="maximumSize">
			<size>
				<width>25</width>
				<height>20</height>
			</size>
			</property>
	
			<property name="text">
			<string>0</string>
			</property>
		</widget>
		</item>

		<item>
		<widget class="QLabel">
			<property name="text">
			<string>Max moves</string>
			</property>
		</widget>
		</item>>

		<item>
		<widget class="QLineEdit" name="maxMovesEditor">
			<property name="maximumSize">
			<size>
				<width>25</width>
				<height>20</height>
			</size>
			</property>

			<property name="text">
			<string>20</string>
			</property>
		</widget>
		</item>
	</layout>
</item>
</layout>


</widget>
</ui>
